install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -31,6 +31,13 @@ podAntiAffinityTermLabelSelector: []
 jaeger:
   hub: docker.io/jaegertracing
   tag: 1.9
+  resources:
+    limits:
+      cpu: 300m
+      memory: 900Mi
```

Comment:
not sure how much jaeger uses normally but this seems pretty low

Reply:
I have a question in jaeger-operator. let us wait for the jaeger experts' answer - https://github.com/jaegertracing/jaeger-operator/issues/562. Thanks.

Reply:
@howardjohn I have open an issue to use badger local storage by default - https://github.com/istio/istio/issues/15914. and please refer to https://github.com/kyma-project/kyma/blob/c547a6bbb62aa28d824c29bbf1d322b54eb4adc9/resources/jaeger/values.yaml#L32:L36 it seems limits `900Mi` is reasonable for `50000` traces. WDYT?

Reply:
My thoughts here is that it is much better to have a limit that is too high rather than too low. If the limit is too low, then the deployment gets OOMKilled or heavily throttled. If its too high... then nothing really bad happens.

I have never used jaeger so I don't know if these are very high or not, but they seem like they might be pretty low. Let me see if I can add jaeger to a large cluster and get some numbers